lib/bitstyles_phoenix/showcase.ex

```diff
@@ -154,4 +153,13 @@ defmodule BitstylesPhoenix.Showcase do
       """
     end
   end
+
+  # copy-paste from phoenix_html_helpers because `content_tag` was removed in phoenix_html v4
+  # and we don't want to depend on phoenix_html_helpers
+  defp content_tag(name, content, attrs) when is_list(attrs) do
+    name = to_string(name)
+    {:safe, escaped} = html_escape(content)
+    sorted_attrs = attrs |> Enum.sort() |> attributes_escape() |> elem(1)
+    {:safe, [?<, name, sorted_attrs, ?>, escaped, ?<, ?/, name, ?>]}
```

Comment:
Wouldn't it be possible here to use a `~H` and the https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#dynamic_tag/1 component youfound? After all it should do the same, right? And might be much nicer, since you don't need to deal with safe-buffers.

Reply:
assigns = {name: name, attrs: sorted_attrs, content: content}
~H"""
<dynamic_tag tag={@name} {@attrs}><%= content %></dynamic_tag>
"""

Something like that?

Reply:
Or even easier, since it just seems to be used once for iframes just render a `<iframe {@atts}`

Reply:
Aren't heex templates only available if you `use Phoenix.Component`?

Reply:
you can import `sigil_H`. It is just a macro

Reply:
You can also just 
`use BitstylesPhoenix.Helper.ComponentRendering`

Should have everything you need I think :)

Reply:
But for sure you can ditch the bit on the content, since we anyway have no content in the iframe tag.

Reply:
Ah alright, that worked, thanks 🙂

Reply:
[`9f92878` (#130)](https://github.com/bitcrowd/bitstyles_phoenix/pull/130/commits/9f92878bbadd54f1e2928e7974d9b805fb51dc8e)